src/test/java/teammates/test/cases/browsertests/FeedbackConstSumOptionQuestionUiTest.java

```diff
@@ -204,4 +206,21 @@ public void testDeleteQuestionAction() {
         assertNull(BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 1));
     }
 
+    private void testUiConsistencyForNewQuestion() {
+        ______TS("CONSTSUM-option after CONSTSUM-recipient: Check ui consistency success case");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("CONSTSUM_RECIPIENT");
+        feedbackEditPage.clickDiscardChangesLinkForNewQuestion();
+        feedbackEditPage.waitForConfirmationModalAndClickOk();
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("CONSTSUM_OPTION");
+
+        assertTrue(feedbackEditPage.isElementVisible(By.id("constSumPoints--1")));
+        assertTrue(feedbackEditPage.isElementVisible(By.id("constSumPointsForEachOption--1")));
```

Comment:
The main bug was that the element of id `constSumOption_Option--1` is hidden so there **must** be an assertion check for that.
So add the most important assertion check, which is to see if `constSumOption_Option--1` is visible or not.

Reply:
@tshradheya  I thought as the radio button and the number field both are hidden in the bug, checking one's visibility would mean that the other is present as they are in the same div?

Reply:
Yes I agree but the bug was because `constSumOption_option--1` is hidden so better to test for that.

Even if you look at the DOM, the `style="display:none"`  is a part of `constSumOption_option--1` which casue the whole thing to be hidden

![image](https://user-images.githubusercontent.com/12574756/37249694-208d79f2-2527-11e8-8144-c24a38fd0c22.png)

Reply:
Agree. So, should I just check the visibility for `constSumOption_option--1` or should I do that for all three radio buttons (that is checking the visibility for the first two and checking if the third is hidden or not), to make the test complete?

Reply:
Doing for all three is good as it makes the tests complete. But make sure to have an assertion check for `constSumOption_Option--1` as that was the one hiding its child element

Reply:
Noted. :+1:  I will make sure the element doesn't hide it's child from us.